certification.py

```diff
@@ -212,10 +229,20 @@ def is_valid(self, name):
             if line.required:
                 required_documents.add(line.document_type.id)
 
+        # update required documents in case has not available and substitute
```

Comment:
Do a loop all the documents in case the replaced document was earlier than required and not available

Reply:
Looks like is_valid is not taking into account the expiration date which is wrong...

And in fact, if we take the date into account I think it makes it clearer that we should remove the certification_not_available field...